components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -500,12 +499,7 @@ class DisplayToolbar internal constructor(
             Toolbar.SiteSecurity.INSECURE -> colors.securityIconInsecure
             Toolbar.SiteSecurity.SECURE -> colors.securityIconSecure
         }
-        if (color == Color.TRANSPARENT) {
-            views.securityIndicator.clearColorFilter()
```

Comment:
Could we limit this change to only the affected Android versions?
Also, if we know the api is broken and is used in multiple places in this file we should refactor all the occurrences.

Reply:
Opened https://github.com/mozilla-mobile/android-components/issues/11549 to investigate the other uses of this API.